optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/partitionedsearch/TestdataPartitionSearchEntity.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.core.impl.testdata.domain.partitionedsearch;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.domain.entity.descriptor.EntityDescriptor;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+import org.optaplanner.core.impl.domain.variable.descriptor.GenuineVariableDescriptor;
+
+@PlanningEntity
+public class TestdataPartitionSearchEntity extends TestdataPartitionSearchObject {
```

Comment:
Why do we need separate Testdata objects for this? I think we can reuse TestdataSolution, if we're willing to add PlanningId on TestdataObject.code.
I'd rather create a non-PlanningId testdata object to test the non-planning id code, that way we can reuse the entire set of testdata objecst (normal, extended, 3th party, etc) directly in Partitioned Search tests. And later in multithreaded-incremental score calculation tests.